### PR TITLE
RK-7316 - Adding 'Collection Level' documentation to the breakpoint tasks section

### PR DESCRIPTION
**Pull request by noagoldman**

### CODE REVIEW

File: docs/breakpoints-tasks.md [hunk 1]
```diff
@@ -21,6 +21,32 @@ The regular expressions expected syntax is the [RE2](https://github.com/google/r
...
+Setting a Low collection level will ensure that only variables at the top of the variable tree will be fetched.
+
+
+To ensure that a specific variable is collected despite being deep in the variable tree, use the Collect Variable functionality.
+
+Default limits for each variable:
+| Variable type                | Python | Node.js | Java | DotNet |
+| ---------------------------- | ------ | ------- | ---- | ------ |
+| **String or Buffer size**    | 512B   | 512B    | 512B | 512B   |
+| **Collection size**          | 20     | 20      | 20   | 20     |
```
Comment (LiranLast) [inline docs/breakpoints-tasks.md:36]: Default collection size for node is 10

File: docs/breakpoints-tasks.md [hunk 2]
```diff
@@ -21,6 +21,32 @@ The regular expressions expected syntax is the [RE2](https://github.com/google/r
 
 <img src="/img/screenshots/data_redaction_example_1.png" />
 
+### Collection level
+
+Collection level is set to minimize the performance impact of collecting variables in a frame that has complex objects, huge strings and infinite reference loops.
+Setting a Low collection level will ensure that only variables at the top of the variable tree will be fetched.
```
Comment (LiranLast) [inline docs/breakpoints-tasks.md:27]: This feels out of context. Only `low`, but not `high` or `medium`

File: docs/breakpoints-tasks.md [hunk 3]
```diff
@@ -21,6 +21,32 @@ The regular expressions expected syntax is the [RE2](https://github.com/google/r
...
+Default limits for each variable:
+| Variable type                | Python | Node.js | Java | DotNet |
+| ---------------------------- | ------ | ------- | ---- | ------ |
+| **String or Buffer size**    | 512B   | 512B    | 512B | 512B   |
+| **Collection size**          | 20     | 20      | 20   | 20     |
+| **Object level**             | 3      | 3       | 4    | 3      |
+| **Collection level**         | 2      | 2       | 2    | 2      |
+
+
+When the Collection level is low, and you want to collect a variable in a higher level, you can:
```
Comment (LiranLast) [inline docs/breakpoints-tasks.md:41]: Again, what does `low` mean?

File: docs/breakpoints-tasks.md [hunk 4]
```diff
@@ -21,6 +21,32 @@ The regular expressions expected syntax is the [RE2](https://github.com/google/r
 
 <img src="/img/screenshots/data_redaction_example_1.png" />
 
+### Collection level
+
+Collection level is set to minimize the performance impact of collecting variables in a frame that has complex objects, huge strings and infinite reference loops.
```
Comment (LiranLast) [inline docs/breakpoints-tasks.md:26]: Great start :)

**Review by yonitwis — APPROVED** [review]

**Pull request merged**